media/css/protocol/components/_footer.scss

```diff
@@ -202,23 +204,24 @@
     .c-footer-section,
     .c-footer-section-social {
         @include bidi(((float, left, right),));
-        padding: 0 ($layout-md / 2);
+        padding: 0 ($layout-md * 0.5);
 
         &:first-child {
-            @include bidi(((padding, 0 ($layout-md / 2) 0 0, 0 0 0 ($layout-md / 2)),));
+            @include bidi(((padding, 0 ($layout-md * 0.5) 0 0, 0 0 0 ($layout-md * 0.5)),));
         }
 
         &:last-child {
-            @include bidi(((padding, 0 0 0 ($layout-md / 2), 0 ($layout-md / 2) 0 0),));
+            @include bidi(((padding, 0 0 0 ($layout-md * 0.5), 0 ($layout-md * 0.5) 0 0),));
         }
     }
 
     @for $cols from 2 through 6 {
-        $width: 100% / $cols;
+        $width: math.div(100%, $cols); // each width is the total width divided by number of columns
+        $gutter: math.div($layout-md, $cols); // each gutter is the base gutter divided by number of columns
         // http://lea.verou.me/2011/01/styling-children-based-on-their-number-with-css3/
         .c-footer-section:first-child:nth-last-child(n+#{$cols}),
         .c-footer-section:first-child:nth-last-child(n+#{$cols}) ~ .c-footer-section:nth-child(1n-1) {
-            width: calc(#{$width} - (#{$layout-md} - (#{$layout-md} / #{$cols})));
+            width: calc(#{$width} - (#{$layout-md} - #{$gutter}));
```

Comment:
matches fix in protocol: https://github.com/mozilla/protocol/pull/726/files#diff-7733f979d2b534258d78ca9d78ffdeb6cb17a5a3a0b8a6f05df1c7f5d3e2eb76R128